worker/draft.go

```diff
@@ -834,7 +834,7 @@ func (n *node) commitOrAbort(pkey uint64, delta *pb.OracleDelta) error {
 		if err != nil {
 			glog.Errorf("Error while applying txn status to disk (%d -> %d): %v",
 				start, commit, err)
-			panic(err)
```

Comment:
we know err is not nil, we can just panic here. Is there a benefit to call x.Panic here?

Reply:
Yeah, keeping `panic()` should be better.